packages/graphiql/test/dev-server.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
for example, if we don't provide a new webpack dev server with the same feature parity as the browserify dev server, then this PR isn't complete